packages/php-wasm/universal/src/lib/emscripten-types.ts

```diff
@@ -0,0 +1,535 @@
+/* eslint-disable @typescript-eslint/no-misused-new */
+/* eslint-disable @typescript-eslint/no-namespace */
+/** Other WebAssembly declarations, for compatibility with older versions of Typescript */
+
+export namespace Emscripten {
```

Comment:
DefinitelyTyped types didn't play well the way we build `rollup.d.ts` – it's easier to ship a copy of these types.